conf/modules.config

```diff
@@ -728,13 +728,12 @@ process{
         ]
     }
 
-    withName: 'MPILEUP_.*' {
-        ext.prefix       = { meta.num_intervals <= 1 ? meta.id : "${meta.id}_${intervals.simpleName}"}
-        ext.when         = { params.tools && params.tools.contains('controlfreec') }
+    withName: 'SAMTOOLS_MPILEUP' {
+        ext.when         = { params.tools && (params.tools.contains('controlfreec') || params.tools.contains('mpileup.gz')) }
         publishDir       = [
             mode: params.publish_dir_mode,
-            path: { "${params.outdir}/variant_calling/${meta.id}/controlfreec" },
-            pattern: "*mpileup",
+            path: { "${params.outdir}/variant_calling/${meta.id}/mpileup.gz" },
```

Comment:
why would you name the outdir `.gz` I would just keep it as `mpileup`

Reply:
oops yes, I used the replacement function and apparently replaced too much...